docs/quickstart.md

```diff
@@ -18,38 +18,16 @@ The environment variable `GENESTACK_PRODUCT` is used to bootstrap specific confi
 It is persisted at /etc/genestack/product` for subsequent executions, it only has to be used once.
 
 ``` shell
-GENESTACK_PRODUCT=openstack-enterprise
+export GENESTACK_PRODUCT=openstack-enterprise
```

Comment:
i think we should change this to `openstack-flex` given it's the only functional option at this point.